src/inmanta/moduletool.py

```diff
@@ -562,6 +663,48 @@ def modules_parser_config(cls, parser: ArgumentParser) -> None:
 
         subparser.add_parser("v1tov2", help="Convert a V1 module to a V2 module in place")
 
+        release = subparser.add_parser(
+            "release",
+            help="Release a new stable or dev release for this module.",
+            description="""
+When a stable release is done, this command:
+* Does a commit that changes the current version to a stable version.
+* Adds Git release tag.
+* Does a commit that changes the current version to a development version that is one patch increment ahead of the released
+  version.
+When a development release is done using the --dev option, this command:
+* Does a commit that updates the current version and the version mentioned in the change to a development version that is
+  a patch, minor or major version ahead of the previous stable release. Whether a patch, minor or major version is created,
+  is determined respectively by the --patch, --minor or --major argument (--patch is the default).
+            """.strip(),
+        )
+        release.add_argument(
+            "--dev",
+            dest="dev",
+            help="Create a development version. The new version number will have the .dev0 build tag.",
+            action="store_true",
+            default=False,
+        )
+        release.add_argument(
+            "--major",
+            dest="major",
+            help="Do a major version bump compared to the previous stable release. Ignored when --dev is not set.",
+            action="store_true",
+        )
+        release.add_argument(
+            "--minor",
+            dest="minor",
+            help="Do a minor version bump compared to the previous stable release. Ignored when --dev is not set.",
+            action="store_true",
+        )
+        release.add_argument(
+            "--patch",
+            dest="patch",
+            help="Do a patch version bump compared to the previous stable release. Ignored when --dev is not set.",
+            action="store_true",
+        )
+        release.add_argument("-m", "--message", help="Commit message", required=True)
```

Comment:
Do we still want to require this? For `inmanta module commit` the purpose was really to commit a change, part of which was the version bump. For `inmanta module release` it's usually really nothing more than a release or a version bump (like what `bumpversion` does). I think a default message would make sense.